### ISSUE TITLE
Init.ps1 can't find NuProj package

### ISSUE THREAD

**Issue by asherber** [AArnott/Nerdbank.GitVersioning#98]
Running `Init.ps1` on a clean clone of the repo gives the following error:
```
PS> .\init.ps1
Restoring NuGet packages...
Unable to find version '0.10.30-beta-gcbd7693f91' of package 'NuProj'.
Restoring NPM packages...
Restoring Typings...
typings WARN deprecated 4/28/2016: "registry:dt/camel-case#0.0.0+20160316155526" is deprecated (updated, replaced or rem
oved)
typings WARN deprecated 11/21/2016: "registry:dt/node#6.0.0+20160915134512" is deprecated (updated, replaced or removed)


+-- del
| `-- pinkie-promise
|   `-- es6-promise
+-- request@2.69.0
| +-- form-data
| `-- tough-cookie
+-- camel-case (global)
`-- node (global)

P:\visual studio\nerdbank.gitversioning
PS>
```

**Comment by asherber** [comment]
It's a NuGet version thing. The laptop I'm on had NuGet 2.8; once I updated to 3.4, the package pulled in correctly. Maybe add this requirement to `CONTRIBUTING.md`?

**Comment by AArnott** [comment]
Why and how were you using 2.8? Init.ps1 is hard coded to use a specific version that it downloads.

**Comment by AArnott** [comment]
Oh snap. It's not. I need to fix that.

**Comment by AArnott** [comment]
Fixed by 4e4bc2da
